src/flow/src/batching_mode/engine.rs

```diff
@@ -361,6 +361,34 @@ impl BatchingEngine {
             }
         }
 
+        let Some(query_ctx) = query_ctx else {
+            UnexpectedSnafu {
+                reason: "Query context is None".to_string(),
+            }
+            .fail()?
+        };
+        let query_ctx = Arc::new(query_ctx);
+
+        // optionally set a eval interval for the flow
+
+        let flow_eval_interval_ms: Option<u128> = flow_options
+            .get(FLOW_EVAL_INTERVAL_KEY)
+            .map(|v| v.parse::<u128>())
+            .transpose()
+            .map_err(|e| {
+                UnexpectedSnafu {
+                    reason: format!("Failed to parse flow eval interval: {e}"),
+                }
+                .build()
+            })?;
```

Comment:
Maybe second is enough? I think there won't have and should not have millisecond interval. Then u64 should be enough.